apps/prairielearn/src/lib/features/index.ts

```diff
@@ -6,7 +6,7 @@ const featureNames = [
   // Should only be applied to courses/institutions.
   'process-questions-in-worker',
   'question-sharing',
-  'bot-grading',
+  'ai-grading',
```

Comment:
Remind me to re-enable this for your course once this deploys; we don't have anything to handle renaming feature flags in a clever/automatic way.